src/pages/workspace/expensifyCard/WorkspaceEditCardLimitPage.tsx

```diff
@@ -0,0 +1,153 @@
+    const [isConfirmModalVisible, setIsConfirmModalVisible] = useState(false);
+
+    const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${policyID}_${CONST.EXPENSIFY_CARD.BANK}`);
+    const card = cardsList?.[cardID] ?? mockedCard;
+
+    const getPromptTextKey = useMemo((): ConfirmationWarningTranslationPaths => {
+        switch (card.nameValuePairs?.limitType) {
+            case CONST.EXPENSIFY_CARD.LIMIT_TYPES.SMART:
+                return 'workspace.expensifyCard.smartLimitWarning';
+            case CONST.EXPENSIFY_CARD.LIMIT_TYPES.FIXED:
+                return 'workspace.expensifyCard.fixedLimitWarning';
+            case CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY:
+                return 'workspace.expensifyCard.monthlyLimitWarning';
+            default:
+                return 'workspace.expensifyCard.fixedLimitWarning';
+        }
+    }, [card.nameValuePairs?.limitType]);
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const updateCardLimit = (newLimit: string) => {
+        setIsConfirmModalVisible(false);
+
+        // TODO: add API call when it's supported https://github.com/Expensify/Expensify/issues/407831
+
+        Navigation.goBack();
+    };
+
+    const submit = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.EDIT_EXPENSIFY_CARD_LIMIT_FORM>) => {
+        const currentLimit = card.nameValuePairs?.limit ?? 0;
+        const currentSpend = currentLimit - card.availableSpend;
+        const newLimit = Number(values[INPUT_IDS.LIMIT]) * 100;
+        const newAvailableSpend = newLimit - currentSpend;
+
+        if (newAvailableSpend <= 0) {
+            setIsConfirmModalVisible(true);
+            return;
+        }
+
+        updateCardLimit(values[INPUT_IDS.LIMIT]);
+    };
+
+    const validate = useCallback(
+        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.EDIT_EXPENSIFY_CARD_LIMIT_FORM>): FormInputErrors<typeof ONYXKEYS.FORMS.EDIT_EXPENSIFY_CARD_LIMIT_FORM> => {
+            const errors = ValidationUtils.getFieldRequiredErrors(values, [INPUT_IDS.LIMIT]);
+
+            // We only want integers to be sent as the limit
+            if (!Number(values.limit) || !Number.isInteger(Number(values.limit))) {
+                errors.limit = translate('iou.error.invalidAmount');
+            }
+
+            return errors;
+        },
+        [translate],
+    );
+
+    return (
+        <AccessOrNotFoundWrapper
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            policyID={policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_EXPENSIFY_CARDS_ENABLED}
+        >
+            <ScreenWrapper
+                testID={WorkspaceEditCardLimitPage.displayName}
+                shouldEnablePickerAvoiding={false}
+                shouldEnableMaxHeight
+            >
+                <HeaderWithBackButton title={translate('workspace.expensifyCard.cardLimit')} />
+                <FormProvider
```

Comment:
<img width="1512" alt="Screenshot 2024-07-18 at 16 31 09" src="https://github.com/user-attachments/assets/cd8c0afa-197f-4d7e-b0c0-34f45e794d7d">

@VickyStash Two error messages. Please add "shouldHideFixErrorsAlert" props

Reply:
> @VickyStash Two error messages. Please add "shouldHideFixErrorsAlert" props

Done - https://github.com/Expensify/App/pull/45397/commits/83e1705dc8a98cf08b20c8eeb3d8450952bf4b30

Reply:
@VickyStash 
> Could you record a video what do you mean?

https://github.com/user-attachments/assets/f2cf2dca-ef84-4c79-87a6-2f9d1b186662

Reply:
@DylanDylann It looks like it's the default form behavior
https://github.com/Expensify/App/blob/f36b308901d6eea9441de411cf3ec5272e61aded/src/components/Form/FormProvider.tsx#L86

But I can turn this param to `false` if it's a more expected result to you.

Reply:
@VickyStash Oh sorry, Let's keep it